client/app/index.html

```diff
@@ -171,7 +171,7 @@
     <script src="node_modules/superdesk-core/scripts/superdesk-authoring/versioning/versioning.js"></script>
     <script src="node_modules/superdesk-core/scripts/superdesk-authoring/versioning/versions/versions.js"></script>
     <script src="node_modules/superdesk-core/scripts/superdesk-authoring/versioning/history/history.js"></script>
-    <script src="node_modules/superdesk-core/scripts/superdesk-authoring/metadata/metadata.js"></script>
+    <script src="scripts/superdesk-authoring/metadata/metadata.js"></script>
```

Comment:
it works locally for me, but changed the path to app/scripts/..., and it still works, grunt build gives no errors.